src/Panel/Panel.d.ts

```diff
@@ -0,0 +1,37 @@
+import { PureComponent, ReactNode } from 'react';
+
+export namespace PanelProps {
+
+  type HeaderSize = 'tiny' | 'small' | 'medium';
+
+  type CollapseDirection = 'up' | 'left' | 'down' | 'right';
+
+  type Header = {
+    collapse?: {
+      direction: CollapseDirection,
+      onExpand: () => void,
+      onCollapse: () => void,
+      isCollapsed?: boolean
+    },
+    size?: HeaderSize,
+    content?: any,
+    title?: ReactNode | string, // TODO(typo): wtf
```

Comment:
mmm `ReactNode` should accept `string`s, can you check this again?

Also, why isn't this `any` like `content` and `menu`?